app/views/static/home.html.haml

```diff
@@ -1,3 +1 @@
 %h1 Welcome to Team Manager!
-%br
-%p Click on the top right of the page to sign in!
```

Comment:
Users will still need to be prompted to sign in to use the website. You may want to say "Sign in with your stuypulse.com account to use Team Manager" or something similar